lib/scrabble/player.rb

```diff
@@ -42,5 +44,15 @@ def highest_scoring_word(word_array)
 		def highest_word_score
 			return Scrabble.score(@highest_score_word)
 		end
+
+		def draw_tiles(tile_bag)
+			@bag = tile_bag
```

Comment:
It doesn't look like you use `@bag` elsewhere in this class, and there's no accessor for it, so you probably don't need to bother keeping it as an instance variable.